sdk/identity/azure-identity/tests/test_managed_identity.py

```diff
@@ -2,6 +2,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
+from azure.identity._credentials import managed_identity
```

Comment:
Was this auto-imported by accident? I know VS Code has done this to me a few times

Reply:
I think so. It's not a line of code I would write :suspect: